<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewController</class>
 <widget class="QWidget" name="ViewController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lRotX">
              <property name="text">
               <string>Поворот отн. oX =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lRotY">
              <property name="text">
               <string>Поворот отн. oY =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lRotZ">
              <property name="text">
               <string>Поворот отн. oZ =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lScale">
              <property name="text">
               <string>Масштаб =           x</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QSlider" name="slRotX">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
                <property name="tickInterval">
                 <number>15</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDial" name="dRotX">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QSpinBox" name="sbRotX">
                <property name="maximum">
                 <number>359</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSlider" name="slRotY">
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
                <property name="tickInterval">
                 <number>15</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDial" name="dRotY">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="sbRotY">
                <property name="maximum">
                 <number>359</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QSlider" name="slRotZ">
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
                <property name="tickInterval">
                 <number>15</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDial" name="dRotZ">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QSpinBox" name="sbRotZ">
                <property name="maximum">
                 <number>359</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QDoubleSpinBox" name="sbScaleMult">
                <property name="maximumSize">
                 <size>
                  <width>55</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.250000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="slScale">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbScale">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_13">
          <item>
           <widget class="QLabel" name="lIsoLevel">
            <property name="text">
             <string>Уров. изопов. =     x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbIsoLevelMult">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.250000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slIsoLevel">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbIsoLevel">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_12">
          <item>
           <widget class="QLabel" name="lGridXDim">
            <property name="text">
             <string>Разм. сетки по oX =   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slGridXDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbGridXDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_16">
          <item>
           <widget class="QLabel" name="lGridYDim">
            <property name="text">
             <string>Разм. сетки по oY =   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slGridYDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbGridYDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_15">
          <item>
           <widget class="QLabel" name="lGridZDim">
            <property name="text">
             <string>Разм. сетки по oZ =   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slGridZDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbGridZDim">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="_14">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rbFaces">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Грани</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="rbEdges">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Ребра</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chSyncGridDim">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Синх. разм. сетки</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="rbNormalFaces">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Норм. грани</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="rbNormalVertexes">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Норм. вершины</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="bShine">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Блеск</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chAxisesShow">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Оси</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
